### PR TITLE
add readme for unstructured data

### PR DESCRIPTION
**Pull request by amy-chalk**

### CODE REVIEW

File: unstructured_data/README.md [hunk 1]
```diff
@@ -1 +1,145 @@
-## Integrating with unstructured data
+# Integrating with unstructured data
+
+Chalk can take unstructured data from sources such as Gemini, ChatGPT, Claude
+and any other APIs you rely on. You can convert this data into features that
+will live along with the rest of your feature codebase, making it simple to
+build models combining structured and unstructured data.
+
+In this example, you will learn:
+- How to use an LLM to classify your card transaction data to determine its
```
Comment (melrchen) [inline unstructured_data/README.md:9]: nit: 
if we're doing "In this example you will" and the other two bullet points are "Define" I think we should move the verb down so it's

In this example, you will: 
- Learn how to..
- Define....
- Define....

Comment (melrchen) [reply unstructured_data/README.md:9]: or 

in this example you will learn how to: 
- use
- define
- define

Comment (amy-chalk) [inline unstructured_data/README.md:9]:
```suggestion
In this example, you will:
- Learn how to use an LLM to classify your card transaction data to determine its
```

Comment (amy-chalk) [reply unstructured_data/README.md]: done!

**Review by melrchen — APPROVED** [review]
⚡

**Pull request merged**